.circleci/config.yml

```diff
@@ -0,0 +1 @@
+version: 2
```

Comment:
Does CircleCI not need more config than this? What about testing multiple Node versions?